src/gui/Src/Gui/CPUStack.cpp

```diff
@@ -1005,7 +1005,7 @@ void CPUStack::dbgStateChangedSlot(DBGSTATE state)
 
 void CPUStack::followInMemoryMapSlot()
 {
-    DbgCmdExec(QString("memmapdump %1").arg(ToHexString(rvaToVa(getInitialSelection()))).toUtf8().constData());
+    DbgCmdExec(QString("memmapdump [%1]").arg(ToHexString(rvaToVa(getInitialSelection()))).toUtf8().constData());

```

Comment:
I think this behavior is better, but it should then replace `Follow in Memory Map` with `Follow DWORD in Memory Map` and also check if that DWORD is a valid memory address before adding it to the menu.